src/guide/advanced/permissions.md

```diff
@@ -0,0 +1,286 @@
+let alice_xor_id = <Asset as Identifiable>::Id::new(
+    AssetDefinitionId::from_str("xor#test"),
+    AccountId::from_str("alice@test"),
+);
+
+// Create a new `CanBurnUserAssets` permission token
+// that allows burning `alice_xor_id` asset
+let permission_token_to_alice: PermissionToken =
+    burn::CanBurnUserAssets::new(alice_xor_id).into();
+
+// Create an instruction that grants Bob permission to burn `alice_xor_id` asset
+let grant = Instruction::Grant(GrantBox::new(
+    permission_token_to_alice,
+    IdBox::AccountId(bob_id.clone()),
+));
+```
+
+### `CanUnregisterAssetWithDefinition`
+
+With `CanUnregisterAssetWithDefinition` permission token, a user can
+unregister assets with the corresponding asset definition.
+
+```rust
+let alice_id = AccountId::from_str("alice@test");
+let bob_id = AccountId::from_str("bob@test");
+let xor_id = AssetDefinitionId::from_str("xor#test");
+
+// Create a new `CanUnregisterAssetWithDefinition` permission token
+// that allows unregistering `xor_id` asset
+let permission_token_to_alice: PermissionToken =
+    unregister::CanUnregisterAssetWithDefinition::new(xor_id).into();
+
+// Create an instruction that grants Bob permission to unregister `xor_id` asset
+let grant = Instruction::Grant(GrantBox {
+    permission_token_to_alice.into(),
+    IdBox::AccountId(bob_id.clone()).into(),
+});
+```
+
+### `CanTransferUserAssets`
+
+With `CanTransferUserAssets` permission token, a user can transfer the
+specified asset.
+
+```rust
+let alice_id = AccountId::from_str("alice@test");
+let bob_id = AccountId::from_str("bob@test");
+let alice_xor_id = <Asset as Identifiable>::Id::new(
+    AssetDefinitionId::from_str("xor#test"),
+    AccountId::from_str("alice@test"),
+);
+
+// Create a new `CanTransferUserAssets` permission token
+// that allows to transfer `alice_xor_id` asset
+let permission_token_to_alice: PermissionToken =
+    transfer::CanTransferUserAssets::new(alice_xor_id).into();
+
+// Create an instruction that grants Bob permission to transfer `alice_xor_id` asset
+let grant = Instruction::Grant(GrantBox::new(
+    permission_token_to_alice,
+    IdBox::AccountId(bob_id.clone()),
+));
+```
+
+### `CanTransferOnlyFixedNumberOfTimesPerPeriod`
+
+With `CanTransferOnlyFixedNumberOfTimesPerPeriod` permission token, a user
+can transfer assets only a fixed number of times per some time period.
+
+```rust
+const PERIOD_MS: u64 = 5000;
+const COUNT: u32 = 2;
+
+// Create a new `CanTransferOnlyFixedNumberOfTimesPerPeriod` permission token
+// that limits the number of transfer to 2 per 5000 ms.
+let permission_token =
+    transfer::CanTransferOnlyFixedNumberOfTimesPerPeriod::new(PERIOD_MS.into(), COUNT);
+```
+
+### `CanSetKeyValueInUserAssets`
+
+With `CanSetKeyValueInUserAssets` permission token, a user can set key
+value in the specified asset.
+
+### `CanRemoveKeyValueInUserAssets`
+
+With `CanRemoveKeyValueInUserAssets` permission token, a user can remove
+key value in the specified asset.
```

Comment:
This two don't have any examples. Is it ok?

Reply:
I've added the missing examples